### DISCUSSION TITLE
How do I add the Prefer header when I do a delta request?

### DISCUSSION THREAD

**Discussion by catmanjan** [microsoftgraph/msgraph-sdk-dotnet#2629] [Q&A]
![image](https://github.com/user-attachments/assets/862e44f9-6380-4683-bd18-09538d55682e)

How do I do this via the SDK?

**Comment by andrueastman** [comment]
Thanks for raising this @catmanjan 

Any chance this doc is useful? It shows how to set the headers in the SDK.

https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/main/docs/headers.md

**Reply by catmanjan** [reply]
Thanks, where would I ask about the business rules behind this particular header? 

I notice if I set it to 10,000 I get 510 emails back

**Reply by andrueastman** [reply]
With regards to the functionality of the API, you can post a question at https://aka.ms/askGraph
